es_gui/tools/valuation/utilities.py

```diff
@@ -368,6 +368,76 @@ def read_miso_reg_price(fname):
     return RegMCP
 
 ##################################################################################################################
+#///////////////////////////////////////////////////////#
+def read_isone_data_updated(fpath, year, month, nodeid):
```

Comment:
As before, if this updated function can entirely replace the previous version, it should replace it.